<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Assistant UI</string>
  </property>
  <widget class="QPushButton" name="runHere">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute Here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCmd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Command</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runCMD">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute on CMD Prompt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clrButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Executing CMD Commands on Python</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
